src/shared/install.c

```diff
@@ -2328,10 +2318,7 @@ int unit_file_unmask(
 
         r = 0;
         STRV_FOREACH(i, todo) {
-                _cleanup_free_ char *path = NULL;
-                const char *rp;
-
-                path = path_make_absolute(*i, config_path);
+                _cleanup_free_ char *path = path_make_absolute(*i, config_path);
```

Comment:
I am not a fan of mixing such changes into an unrelated commit... Of course, the change itself is ok.

Reply:
I'll drop this from the commit. It's not really relevant.